templates/taxi/driver_confirm_delete.html

```diff
@@ -0,0 +1,10 @@
+{% extends "base.html" %}
+
+{% block content %}
+  <h1>Delete {{ driver.username }}?</h1>
+  <form action="" method="post">
+    {% csrf_token %}
+
+    <input type="submit" value="Yes" class="btn btn-danger">
```

Comment:
maybe you should add "cancel" or "no" button to this page

Reply:
yes thank you I did in some previous task and forget it here